robot/util/rosRoverStart/local_rover.launch

```diff
@@ -2,4 +2,15 @@
     <node pkg="mux_selector" type="mux_select_server.py" name="mux_select_server" output="screen" args="local"/>
     <node pkg="web_video_server" type="web_video_server" name="web_video_server" output="screen"/>    
     <node pkg="task_handler" type="task_handler_server.py" name="task_handler_server" output="screen" args="local"/>
+    <node pkg="mcu_control" type="MockPdsNode.py" name="mock_pds_node" output="screen"/>
+    <param name="PDS_mock_voltage" value="stable"/>
+    <param name="PDS_mock_temp1" value="stable"/>
+    <param name="PDS_mock_temp2" value="stable"/>
+    <param name="PDS_mock_temp3" value="stable"/>
+    <param name="PDS_mock_wheel_current1" value="stable"/>
+    <param name="PDS_mock_wheel_current2" value="stable"/>
+    <param name="PDS_mock_wheel_current3" value="stable"/>
+    <param name="PDS_mock_wheel_current4" value="stable"/>
+    <param name="PDS_mock_wheel_current5" value="stable"/>
+    <param name="PDS_mock_wheel_current6" value="stable"/>
```

Comment:
A few things could be added here.

- The starting value (regardless if you are stable, rising or falling, you should start with the same value. Battery should be 15V, temps 25 and currents I actually have no clue ). The starting value can be 1 param for voltage, 1 for all temps and 1 for all currents.

- Noise value should be a ROS param. I don't think you need a noise value per param. Maybe just make 1 noise value param that applies for everything. `0.05` is a good default imo so that's good

- Rise/Fall rate should also be a param. The rate can be 1 param for voltage, 1 for all temps and 1 for all currents.